### ISSUE TITLE
Unassigned empty player slot

### ISSUE THREAD

**Issue by insunaa** [MNoya/PMP#11]
When a few people enter and leave a lobby in matchmaking, sometimes an empty playerslot passes into the lobby. This is fine in FFA, as there is a 30 second timer that automatically starts the game, but in team mode there is no timer. The empty player cannot be assigned to any team and as long as it is unassigned, the match cannot be started. This cannot be solved and all players have to disconnect from the lobby if that happens.

Possible fix: Implement a timer to auto-start the game on the team map.

Edit: This should not be a bug in PMP, it should be a dota bug, but Valve has so far not been interested in fixing it.

Edit2: This is probably the cause for this: #4 

Edit3: Here's an example of an empty unassigned player in FFA
![bug_2](https://cloud.githubusercontent.com/assets/3718129/15097891/1fc5acf4-152c-11e6-8fda-dcc83a89fe6d.png)
(Excuse the bad gimp skills)

**Comment by MNoya** [comment]
Yeah I totally had this bug happen on other mods aswell. I'll try implementing a check to force the autostart. Thanks for bringing it to my attention!
